src/components/NcAppNavigation/NcAppNavigation.vue

```diff
@@ -4,6 +4,105 @@
 -->
 
 <docs>
+```vue
+<template>
+	<div class="styleguide-nc-content">
+		<NcAppNavigation>
+			<template>
+				<div class="navigation__header">
+					<NcTextField :value.sync="searchValue" label="Search …" />
+					<NcActions>
+						<NcActionButton close-after-click @click="showModal = true">
+							<template #icon>
+								<IconCog />
+							</template>
+							App settings (close after click)
+						</NcActionButton>
+						<NcActionButton @click="showModal = true">
+							<template #icon>
+								<IconCog />
+							</template>
+							App settings (handle only click)
+						</NcActionButton>
+					</NcActions>
+				</div>
```

Comment:
@Antreesy when you handle the `.sync` could you also rephrase this part to not have something like search? Because search should only be handled by `NcAppNavigationSearch` and we should not have examples doing it other

Reply:
- https://github.com/nextcloud-libraries/nextcloud-vue/pull/5848 is not backported yet

Don't you mind, if I forward-port it as-is, then replace this example on master and then push to the next afterwards?